application/dash_application/views/preqc_wgs.py

```diff
@@ -100,6 +100,7 @@
     {"label": "Run", "value": PINERY_COL.SequencerRunName},
     {"label": "Kit", "value": PINERY_COL.PrepKit},
     {"label": "Tissue Prep", "value": PINERY_COL.TissuePreparation},
+    {"label": "Library Design", "value": PINERY_COL.LibrarySourceTemplateType},
```

Comment:
Could Library Design please be removed for this report? By definition, it'll only display WG samples.

Reply:
Done!